fraud/registry.go

```diff
@@ -0,0 +1,11 @@
+package fraud
+
+var DefaultUnmarshalers = map[ProofType]ProofUnmarshaler{}
```

Comment:
It's better to be protected with a lock and thus encapsulated behind a getter, like Register is for writes

Reply:
done